bin/node/runtime/src/lib.rs

```diff
@@ -58,6 +62,7 @@ use sp_api::impl_runtime_apis;
 use sp_authority_discovery::AuthorityId as AuthorityDiscoveryId;
 use sp_core::{crypto::KeyTypeId, OpaqueMetadata};
 use sp_inherents::{CheckInherentsResult, InherentData};
+use sp_io::hashing::blake2_128;
```

Comment:
Should we touch it for the requred update?

Reply:
It was due to cargo fmt command

Reply:
I would apply it to the new code only to keep the "blame" history and simplify future merges with upstream.

Reply:
@Raid5594 
I agree. We should apply formatting only to code we've added. 
If we have a need to run fmt for the whole repo we should do it in a separate commit and PR.